TODO.md

```diff
@@ -0,0 +1,26 @@
+## TODO
+
+* Add datapoints in provisioning logic for Influx
+* Be more specific in JSON Schema
+* Load things from NextToken when retreiving spot price history
+* Overwrite UserData with temporary taskcluster credentials
+  * Include the API endpoint for the slave to hit the hello
+    endpoint
+  * include the instanceId scope!
+* Encrypt UserData using opengpg.js
+* Send pulse messages for
+  * killing an instance
+  * cancelling a spot request
+  * creating a spot request
+* Use Joi/assert to better control the input into functions
+* Verify that errors in provisioning don't kill the entire
+  provisioner
+* Kill capacity which exceeds maxCapacity
```

Comment:
Yep, I'm writing that today but it's not going to change any major architectural things.